.github/workflows/build.yml

```diff
@@ -0,0 +1,135 @@
+name: continuous integration
+
+on: [push, pull_request]
+
+jobs:
+  build_whole_workspace:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-2019, macOS-10.14]
+        rust: [stable, nightly]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+      - name: Install rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+      - name: Build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
```

Comment:
this can be a `check` instead of `build` which should speed up CI time.

Reply:
fixed